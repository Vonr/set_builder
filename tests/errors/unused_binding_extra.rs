use set_builder::set;

fn main() {
    let _ = set! { (x, y) : x <- [1, 2, 3] };
}
