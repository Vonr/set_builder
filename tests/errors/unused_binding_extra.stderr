error: proc macro panicked
 --> tests/errors/unused_binding_extra.rs:4:13
  |
4 |     let _ = set! { (x, y) : x <- [1, 2, 3] };
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: y is not bound to any sets
